app/services/badges/award_community_wellness.rb

```diff
@@ -46,7 +46,8 @@ def self.call
         else
           # If the FeatureFlag isn't enabled only track which users would get
           # the badge to get an understanding of how the service will work
-          Ahoy.instance&.track("Community Wellness Badge Award", user_id: user.id, weeks: week_streak)
+          tags = ["user_id:#{user.id}", "weeks:#{week_streak}"]
+          ForemStatsClient.increment("consumer_apps.create", tags: tags)
```

Comment:
Whoops my bad here! Copy/paste almost gone wrong here 😳

```suggestion
          ForemStatsClient.increment("community_wellness_badge.award", tags: tags)
```

Merging the PR after this change goes in 👍